packaging/android/build_and_publish.sh

```diff
@@ -36,6 +40,14 @@ function reset {
 }
 trap reset EXIT
 
+if [ -n "$kbfs_commit" ]; then
+  cd "$kbfs_dir"
+  echo "Checking out $kbfs_commit on client (will reset to $kbfs_branch)"
```

Comment:
the message should be `Checking out $kbfs_commit on kbfs` (I mentioned this previously; did you have a reason for not changing it?)